views/pages/NewConfigPage.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { useForm } from 'react-hook-form'
+import { useOutletContext } from 'react-router-dom'
+
+import api from '../api'
+import useArrayFormFields from '../hooks/useArrayFormFields'
+import useGrid from '../hooks/useGrid'
+import ConfigForm from '../forms/ConfigForm'
+import { UserConfigModel, UsersModel } from '../models'
+import { colorList } from '../fe-utils/colorList'
+
+const colors = colorList().value()
```

Comment:
Would it make sense to have `colorList` return the unwrapped lodash object so that we don't have to call `value` on it?

Reply:
[18d2bf3](https://github.com/AMP-SCZ/dpdash/pull/332/commits/18d2bf32b1522b46ecd8403f4c93f0760399d9cb)